app/assets/images/svg/vendor-ibm.svg

```diff
@@ -0,0 +1 @@
CGa5B593u4w2QN9pxCiafSDbLEo9gnvs6xC3ASp8LnSo54A4maIdfgSqr+Lm+GQCC/hxc3gdfbctkK/eGwNYG1mnyrcR4gmmEppSuVJIZrljuT3MYzqjwH8D32ETzchmiW+eSxSyuslvGia+JtWoJd7VWhrACuaJid8GsX9c7oEQYjmSFOanNxrSHZxI6QR3uWOKKURN4OvcS/V4FfJsbWZ6d3wollyKU0azeMxIZomvlsyFT33ow+JL5rGhc+JHas1PmjVcmHRNHcAQPLX6H/Ol8vnvQ1sZmKFaJL4w+zsgjGMpV+b2YcgRIPEv+gRS5IMW/pZg1jRLPuE95w9Cp+r1QvRBLuUhuRryKcwj7fbKuyFaJL05UpPsJF6vW1H+Z+if0fNyszeCyEaI/2rfJ6ulPL0mPJ41UaIZhhEeJJ/wfDdkvEFq2v0ZUvfC9EUufdFRrnT2nz83JnZ85nvC1EtuT80HKM60EfyDv07KH3Zgao1oilKwqdypymO5/oSXjSFkWNnzewFjKO5hfbgHfIk/z26iRAVkovwwFB0JO0Ow+NCNENJ+C6zJ4ZRH5DwojGyKQ0AmNlzGMudir7L7Um+ASEqpxThgXG1JhfVFelFU0z93aY4OI0bMsdoZs/mbiJETRSFJ/kfbyZb2hHSJ4EQ1bLvL/ONIjlC3o7kCWBmb83dRIhamBSe5OsoR/NcJxCiavZFeMdlj+24h583s2dG3xaiEvYKT/K/GNfjU9G9rWgvqmav8FvSwaofS9sEADObKncKcTUOEp7km/Ej8rILUT0HCb9l3wB1MIg1M72wSVRHcWlBjjAgTb8UP8fID5Ke/wtxdU7JtXNyT7WFqIajIjyQHZDuE5089h8R4kwck8OnHDJw9aqNZc4JcXGOFp5k+gqPkuiK6qI6jhY+MBnVUxTlRQ2cJDzJ3Dtp9kZ1SS+uzSlVmhSlLqIZTorwwK6+LtlFUxxdlkw5NE1RaVLUwMkR/hgku6iFBws/JbNEF7Xx4JTGSVMbyS5q5MER3omCS3ZRK7MJ70h2UTOzpTRCtMDsEV6ImpHwYlFIeLEoJLxYFBJeLAoJLxaFhBeLQsKLRSHhxaKQ8GJRSHixKCS8WBQSXiwKCS8WhYQXi0LCi0Uh4cWikPBiUUh4sSgkvFgUEl4sCgkvFoWEF4tCwotFIeHFopDwYlFIeLEoJLxYFBJeLAoJLxaFhBeLQsKLRSHhxaKQ8GJRSHixKP4PM6F2KLxHKdQAAAAASUVORK5CYII="/></g></mask><linearGradient id="linear-gradient" x1="210.3" y1="17.18" x2="76.91" y2="243.57" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#3ddbd9"/><stop offset="1" stop-color="#4589ff"/></linearGradient></defs><title>Asset 1@12x-100</title><g id="Layer_2" data-name="Layer 2"><g id="artwork"><g class="cls-1"><g id="IBM_Cloud_logo" data-name="IBM Cloud logo"><path class="cls-2" d="M9.6,82.26,34.51,96.64a5.19,5.19,0,0,0,5.19-9L14.79,73.26a5.19,5.19,0,0,0-5.19,9Z"/><path class="cls-2" d="M75.2,52.14a5.19,5.19,0,1,0,9-5.19L69.82,22a5.19,5.19,0,0,0-9,5.19Z"/><path class="cls-2" d="M137.89,39.15A5.19,5.19,0,0,0,143.08,34V5.19a5.2,5.2,0,0,0-10.39,0V34A5.19,5.19,0,0,0,137.89,39.15Z"/><path class="cls-2" d="M193.48,54.05a5.2,5.2,0,0,0,7.09-1.91L215,27.23A5.19,5.19,0,0,0,206,22L191.58,47A5.19,5.19,0,0,0,193.48,54.05Z"/><path class="cls-2" d="M238.67,97.34a5.23,5.23,0,0,0,2.6-.7l24.91-14.38a5.2,5.2,0,1,0-5.2-9L236.07,87.65a5.19,5.19,0,0,0,2.6,9.69Z"/><path class="cls-2" d="M218.34,102.31A94.52,94.52,0,0,0,204.11,84.1,93.71,93.71,0,0,0,47.87,124.35,60.48,60.48,0,0,0,60,244H204.24a71.54,71.54,0,0,0,14.1-141.68ZM79,91.45a83.27,83.27,0,0,1,117.76,0,84.59,84.59,0,0,1,8.16,9.46h-.16a71.55,71.55,0,0,0-56.22,116.46l8.08-6.53a61.16,61.16,0,0,1,48.06-99.55,62.56,62.56,0,0,1,7,.46,83.3,83.3,0,0,1-73.8,121.85h-.06a83.27,83.27,0,0,1-83.24-83.27,84,84,0,0,1,1.68-16.75c1.38-.11,2.77-.17,4.18-.17h.38a50.16,50.16,0,0,1,49.72,50.47L121,184a60.55,60.55,0,0,0-60-60.93h-.45l-1.3,0A82.76,82.76,0,0,1,79,91.45ZM60.09,233.6a50.1,50.1,0,0,1-14.73-97.86,95.49,95.49,0,0,0-1.13,14.59,93,93,0,0,0,27.43,66.22,94,94,0,0,0,23.27,17Zm144.15,0h-23.4a93.86,93.86,0,0,0,43.74-118.82A61.15,61.15,0,0,1,204.24,233.6Z"/></g></g></g></g></svg>
```

Comment:
There's still a png encoded in the svg file :confused: 
```
xlink:href="data:image/png;base64
```

Reply:
ok, let me check

Reply:
I've totally forgotten. I've spoken to Jay and as it seems, we're currently not able to get an officially sanctioned **vector** image from our branding team. The idea is to submit this one until we get a vector image in the future.

Reply:
Okay, just create a new PR replacing this file if you get your hands on a proper vector one. I'll merge this after it's green...